incubator/kafka/values.yaml

```diff
@@ -143,52 +143,100 @@ persistence:
   ##
   # storageClass:
 
-## Metrics
+jmx:
+
+  ## Rules to apply to the Kafka JMX Exporter
+  configMap:
+
+    ## Allows disabling the default configmap, note a configMap is needed
+    enabled: true
```

Comment:
This doesn't seem like something that is mean to be under the configMap key?

I think I'm not sure why all of this section seems to be under a `configMap` key at all (rather than just directly under `jmx`.

As well, alot of this seems like it is effectively the same as what was previous under the `metrics` key, just moved?

Reply:
It's intentional, allowing people to disable the configmap and provide there own can be very useful.